plugins/history.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: history
+spec:
+  version: v1.1.8
+  homepage: https://github.com/wd/kubectl-history
+  shortDescription: Plugin to List and diff history reversions of deployment/daemonset/statefulset
```

```suggestion
  shortDescription: List & diff revisions of workload resources
```

please keep this short ~50 chars as it's truncated.